packages/hash/frontend/src/components/hooks/blockProtocolFunctions/knowledge/knowledge-shim.ts

```diff
@@ -31,15 +32,22 @@ type UnsupportedPersistedEntityFields = "linkedEntities";
 
 type DeprecatedPersistedEntityFields = "accountId";
 
-type Entity = Omit<
+type BaseEntity = Omit<
   UnknownPersistedEntity,
   | UnsupportedPersistedEntityFields
   | DeprecatedPersistedEntityFields
   | "entityType"
-> & {
+>;
+
+type Entity = BaseEntity & {
+  links: Link[];
   entityTypeRootedSubgraph: EntityTypeRootedSubgraph;
 };
 
+type Link = Omit<PersistedLink, "sourceEntity" | "targetEntity"> & {
```

Comment:
Oh dear, I'm glad we're revisiting all of these types. Definitely shouldn't have to be doing this sort of thing